browser/src/UI/components/CursorColumn.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import { connect } from "react-redux"
+
+import * as Config from "./../../Config"
+import * as State from "./../State"
+
+export interface ICursorColumnProps {
+    x: number
+    y: number
+    width: number
+    height: number
+    color: string
+    visible: boolean
+}
+
+// require("./Cursor.less") // tslint:disable-line no-var-requires
+
+class CursorColumnRenderer extends React.Component<ICursorColumnProps, void> {
+
+    public render(): null |  JSX.Element {
+        if (!this.props.visible) {
+            return null
+        }
+
+
+        const fontFamily = Config.getValue<string>("editor.fontFamily")
+        const fontSize = Config.getValue<string>("editor.fontSize")
+        const width = this.props.width
+
+        // TODO: Add width
+
+        const cursorStyle = {
+            position: "absolute",
+            left: this.props.x.toString() + "px", // Window Start
+            top: this.props.y.toString() + "px", // Same as cursor
+            width: width.toString() + "px", // Window width
+
+            height: this.props.height.toString() + "px", // Same as cursor
+            backgroundColor: this.props.color,
+            opacity: 0.2,
+            fontFamily,
```

Comment:
I think we can get rid of fontFamily and fontSize here, same for CursorLineRenderer. We could actually use the same `Renderer` class for both the CursorColumn/CursorLine, since there isn't actually anything different about them, except the way we figure out the height/width, which happens in the mapStateToProps